Editor/Linux/Install.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
+
+import os
+
+os.system('clear')
+
+answer_installModuleDependencies = input("This installer has some Python dependencies, would you like to install them? [Yes/No]" + '\n')
+
+if answer_installModuleDependencies == "Yes":
+    print("Installing required modules.")
+    os.system('pip install pillow ttkbootstrap distro')
+elif answer_installModuleDependencies == "No": 
+    print("Skipping module dependencies!" + '\n' + "The installer might not run.")
+elif answer_installModuleDependencies == "Yes":
```

Comment:
this code will never execute? Or am I wrong?

Reply:
Looks like it

Reply:
What do you mean?

Reply:
That the first line of the branch `if answer_installModuleDependencies == "Yes":` is the same as this elif line: `elif answer_installModuleDependencies == "Yes":` So the elif line would never run. You don't need to close the PR because of this, unless you don't want to fix this.